### ISSUE TITLE
text dominant-baseline not working

### ISSUE THREAD

**Issue by belst** [yWorks/svg2pdf.js#80]
**Describe the bug**
I am using `dominant-baseline="central"` to vertically center some text.
This works fine in the svg, but not in the generated pdf.

**Screenshots**
SVG: ![svg](https://0x0.st/zcxe.png)

PDF: ![pdf](https://0x0.st/zcx2.png)

**Desktop (please complete the following information):**
 Chromium Version 74.0.3729.131 (Official Build) Arch Linux (64-Bit)

**Comment by HackbrettXXX** [comment]
You're right, this is not supported by svg2pdf. We don't plan on implementing this by ourselves. We are happy about a pull request, though.
